conf.py

```diff
@@ -156,6 +157,15 @@
 		('/team/', 'Team'),
 		('/donate/', 'Donate'),
     ),
+    "bg": (
+		('/bg/blog/', 'Блог'),
+		('/bg/install/', 'Инсталация'),
+		('/bg/gallery/1.22/catalan/', 'Екранни снимки'),
```

Comment:
Why Catalan languages for screenshots?
Might be better to use the english (default) screenshots.

Reply:
Yeah, i just copy pasted the Catalan one and didn't see that. For now lets use the english one, later I will upload a bulgarian screenshot